elasticsearch/kibana_dashboards/manageiq_ops_overview.json

```diff
@@ -0,0 +1,13 @@
+{
+  "title": "ManageIQ Ops Overview",
+  "hits": 0,
+  "description": "",
+  "panelsJSON": "[{\"id\":\"hostnames\",\"type\":\"visualization\",\"panelIndex\":1,\"size_x\":7,\"size_y\":4,\"col\":1,\"row\":1}]",
+  "optionsJSON": "{\"darkTheme\":true}",
+  "uiStateJSON": "{}",
+  "version": 1,
+  "timeRestore": false,
+  "kibanaSavedObjectMeta": {
+    "searchSourceJSON": "{\"filter\":[{\"query\":{\"query_string\":{\"query\":\"*\",\"analyze_wildcard\":true}}}]}"
+  }
+}
```

Comment:
@enoodle Is this dashboard really making any specific information available? I haven't tried it myself but it looks pretty basic to me. The reason why I ask is that I myself run into issue that standard Kibana dashboards exports do not contain index patterns (which can contain other important data/objects used on dashboards) and index patterns are required to be recreated manually by users. Have you run into similar issue yourself as well?